test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/AnonymizedExportTests.cs

```diff
@@ -55,8 +59,10 @@ public AnonymizedExportTests(ExportTestFixture fixture)
             _exportConfiguration = ((IOptions<ExportJobConfiguration>)(fixture.TestFhirServer as InProcTestFhirServer)?.Server?.Services?.GetService(typeof(IOptions<ExportJobConfiguration>)))?.Value;
         }
 
-        [Fact]
-        public async Task GivenAValidConfigurationWithETag_WhenExportingAnonymizedData_ResourceShouldBeAnonymized()
+        [Theory]
```

Comment:
Can we change this to be a SkippableTheory and use the Skip.If call instead of just returning? It makes it easier to see when the test is actually running. There are some examples in CustomSearchParamTests.

Reply:
Updated anonymized export tests as skippable and use Skip.If to mark if tests is not actually running.